api/orchestration_movers.go

```diff
@@ -453,31 +453,55 @@ func (o *datasyncOrchestrator) taskDetailsFromName(ctx context.Context, group, n
 	return nil, nil, apierror.New(apierror.ErrNotFound, "datasync mover not found", nil)
 }
 
-// taskRunsFromTaskArn finds a datasync task runs  based on its group/name and returns information about it
-func (o *datasyncOrchestrator) taskRunsFromTaskArn(ctx context.Context, taskArn string) ([]string, error) {
-	if taskArn == "" {
+// datamoverRunList finds a datasync task runs  based on its group/name and returns information about it
+func (o *datasyncOrchestrator) datamoverRunList(ctx context.Context, group, name string, fullArn bool) ([]string, error) {
+	respL, errL := o.datamoverDescribe(ctx, group, name)
+
```

Comment:
✂️  no empty line between function calls and err check

Reply:
Done.